var/spack/repos/builtin/packages/hipace/package.py

```diff
@@ -37,10 +39,11 @@ class Hipace(CMakePackage):
         description="Floating point precision (single/double)",
     )
 
+    depends_on("cmake@3.18.0:", type="build", when="@23.07:")
     depends_on("cmake@3.15.0:", type="build")
-    depends_on("cuda@9.2.88:", when="compute=cuda")
     depends_on("mpi", when="+mpi")
     with when("+openpmd"):
+        depends_on("openpmd-api@0.15.1:", when="@23.07:")
```

Comment:
Pointed out by @SeverinDiederichs: v23.05 already requires the latest openPMD-api.
```suggestion
        depends_on("openpmd-api@0.15.1:", when="@23.05:")
```